src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1108,10 +1108,10 @@ trait Contexts { self: Analyzer =>
       //
       // A type-import-on-demand declaration never causes any other declaration to be shadowed.
       //
-      // Scala: Bindings of different kinds have a precedence deﬁned on them:
+      // Scala: Bindings of different kinds have a precedence defined on them:
```

Comment:
Right, I didn't notice it's simply an excerpt from Scala spec.

Btw, it does not matter, but since I declared that I'll look at the history: https://github.com/scala/scala/commit/8383b65fb53ae9424cf9c6f1314ee73321ad3b9a